teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown')
+    super(name, age)
```

Comment:
- [ **OPTIONAL** ] **IMPORTNAT:** Good job adding all the changes mentioned by the previous reviewer. However, to correctly match the inheritance of the `Person` class, kindly make sure you update your `initialize` and `super()` methods by also adding the `parent_permission` attribute as it misses there for now. i.e.
```ruby
...
class Teacher < Person
  def initialize(specialization, age, name = 'Unknown', parent_permission: true)
    super(name, age, parent_permission)
    ...
```